src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,111 @@
-function rottenTomatoes(grid) {}
+//helper function
+
+function findFresh(map) {
+    for (let row = 0; row < map.length; row++) {
+        for (let column = 0; column < map[row].length; column++) {
```

Comment:
You can compute `map.length` and `map[row].length` as variables so you don't have to repeat it throughout.